toolchain/parse/typed_nodes.h

```diff
@@ -1032,6 +1034,51 @@ struct DesignatorExpr {
   NodeIdOneOf<IdentifierName, SelfTypeName> name;
 };
 
+// FIXME: RequirementRewrite?
```

Comment:
I note this is a FIXME not a TODO. Do you want the naming question here addressed before you merge this?

I lean towards using syntactic rather than semantic names for parse tree nodes, so maybe`RequirementEqual` and `RequirementEqualEqual`, matching the `TokenKind` names for the operators.